lib/dragonfly/data_storage/cloudinary_store.rb

```diff
@@ -25,6 +25,26 @@ def url_for(uid, options = {})
         ::Cloudinary::Utils.cloudinary_url(public_id(uid), options)
       end
 
+      # Store the data AND meta, and return a unique string uid
+      def write(content, opts={})
+        store(content, opts)
+      end
+
+      # Retrieve the data and meta as a 2-item array
+      def read(uid)
+        data = retrieve(uid)
+        resource_data = ::Cloudinary::Api.resource(public_id(uid), exif: true)
+        if data
+          [
+              data,     # can be a String, File, Pathname, Tempfile
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.